caffe2/operators/fused_rowwise_8bit_conversion_ops.cc

```diff
@@ -47,9 +47,9 @@ OPERATOR_SCHEMA(Fused8BitRowwiseQuantizedToFloat)
     .SetDoc(R"DOC(
 De-quantizes the result of the
 FloatToFused8BitRowwiseQuantized operator. The input is expected to
-encode the scale as a 32-bit float in the first 4 bytes of each row,
-followed by the bias as a 32-bit float in the next 4 bytes, followed by
-the quantized values in the remaining bytes of the row. The output is a
+encode the scale as a 32-bit float in the second to the last 4 bytes of each
+row, followed by the bias as a 32-bit float in the next 4 bytes, and the
+quantized values in the preceding remaining bytes of the row. The output is a
```

Comment:
"preceding remaining" is a little confusing - do you just mean "preceding" here?

Reply:
Thanks for catching the typo.